extensions/emmet/src/util.ts

```diff
@@ -562,4 +564,35 @@ export function isStyleAttribute(currentNode: Node | null, position: vscode.Posi
 	}
 	const styleAttribute = currentHtmlNode.attributes[index];
 	return position.isAfterOrEqual(styleAttribute.value.start) && position.isBeforeOrEqual(styleAttribute.value.end);
+}
+
+export function isTemplateScript(currentNode: HtmlNode): boolean {
+	return currentNode.name === 'script' &&
+		(currentNode.attributes &&
+			currentNode.attributes.some(x => x.name.toString() === 'type'
+				&& allowedMimeTypesInScriptTag.indexOf(x.value.toString()) > -1));
+}
+
+/**
+ * Given `currentNode` as a script HTML template, returns the `HtmlNode`
+ * at `position` within that template.
+ */
+export function getTemplateScriptNestedNode(document: vscode.TextDocument, currentNode: HtmlNode, position: vscode.Position): HtmlNode | undefined {
+	if (isTemplateScript(currentNode) &&
+		(position.isAfter(currentNode.open.end) && position.isBefore(currentNode.close.start))) {
```

Comment:
Woudlnt this fail on `<script type="text/html"/>`?

Reply:
What do you mean by "fail" here? If it is of type `text/html`, `isTemplateScript` will be true and the code will execute. Is this incorrect?